app/controllers/contact/govuk/accessible_format_requests_controller.rb

```diff
@@ -0,0 +1,107 @@
+require "gds_api/publishing_api"
+
+class Contact::Govuk::AccessibleFormatRequestsController < ContactController
+  helper_method :attachment_title
+  helper_method :content_path
+  before_action :check_content_specified
+
+  rescue_from GdsApi::BaseError, with: :content_item_error
+  rescue_from AttachmentNotFoundError, with: :content_item_error
+
+  layout "accessible_format_requests"
+
+  def format_type
+    @back_path = content_path
+
+    if params[:error] == "format-type-missing"
+      flash.now[:input_errors] = [[I18n.t("controllers.contact.govuk.accessible_format_requests.format_type_error"), "format_type"]]
+      flash.now[:format_type_error] = I18n.t("controllers.contact.govuk.accessible_format_requests.format_type_error")
+    end
+
+    if params[:error] == "other-format-missing"
+      flash.now[:input_errors] = [[I18n.t("controllers.contact.govuk.accessible_format_requests.other_format_error"), "other_format"]]
+      flash.now[:other_format_error] = I18n.t("controllers.contact.govuk.accessible_format_requests.other_format_error")
+    end
+  end
+
+  def contact_details
+    if params[:format_type].blank?
+      redirect_to(contact_govuk_request_accessible_format_path(content_id: params[:content_id], attachment_id: params[:attachment_id], error: "format-type-missing"))
```

Comment:
Hmm. It's not currently possible to get both redirects triggered (they rely on mutually exclusive if conditions), but in fact this was just an oversight, and it would be better to be explicit about the return just in case someone changes the conditions in future. I'll add in the return - I'll also have to move the initialisation of @back_path above these guards, otherwise it'll fail when they're fixed (at the moment it's working because of the bug).